stores/user.ts

```
@@ -53,17 +53,22 @@ export const useUserStore = defineStore("user", {
       eventSlug: string,
       scheduleId: string,
       formResponseId?: string,
-      status?: "able-to-donate" | "unable-to-donate"
+      status?: "able-to-donate" | "unable-to-donate",
+      lastQuestionnairePreScreening?: any
```

ESLint

[error] Insert `,`

(prettier/prettier)